services/src/services/errorTelemetry/errorTelemetry.class.js

```diff
@@ -2,7 +2,7 @@ const fs     = require('fs');
 const logger = require('winston');
 const path   = require('path');
 const mkdirp = require('mkdirp');
-
+const sentry = require('../../lib/sentry');
```

Comment:
The CI error seems to come from this. I seem to understand the `raven` module got imported above, but then the code references a manually (?) added lib from two directories above?

I seem to understand that the line above should just be replaced by 

```js
const sentry = require('raven');
```

And then it would work?